keyboards/omnikeyish/keymaps/default/keymap.c

```diff
@@ -0,0 +1,18 @@
+#include "quantum.h"
+#include "matrix.h"
+#include "config.h"
+#include "dynamic_macro.h"
+
+#define M_PROG DYN_MACRO_PROG
+
+/*    COL1     COL2                          COL3     COL4     COL5     COL6     COL7     COL8     COL9     COL10    COL11    COL12    COL13    COL14    COL15    COL16        COL17    COL18    COL19        COL20    COL21    COL22    COL23 */
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = {
```

Comment:
We don't use the raw matrix anymore.  We use `LAYOUT` macros, so that the keyboard matrix can be reformatted into something human readable.

Reply:
Since there's a near 1:1 equivalence between the matrix and physical layout on THIS pcb, the layout macro was more of a hindrance for me. 🤷‍♂️

Reply:
It can be intimidating, at first, yeah.   In fact, I had issues with it for a long while.   Now, not so much. 

If you want, I can see about posting what it should look like, for you.

Reply:
Added a few LAYOUT macros, there's tens of Northgate plate layouts depending on PCB rev and model. And once you start considering custom or moded plates, the possible layouts goes up over a thousand. 

The default LAYOUT macro let's you configure any legal layout according to the PCB, but the switch plate used with the PCB, will limit one's valid switch positions.

Reply:
I think, just add the most popular 2-5 and then let people deal with it themselves is the usual advice. As long as LAYOUT or LAYOUT_all let's them express themselves they can always submit a PR.

Reply:
An additional small alignment is that the current convention for the LAYOUT macros is lowercase after the `LAYOUT_` part, e.g. `LAYOUT_ultra_t`.

Reply:
Added LAYOUT macros for some northgate models. Based on my two samples, the omnikey blackheart deskthority thread, and google.